cmd/estuary-shuttle/main.go

```diff
@@ -9,6 +9,7 @@ import (
 	"net/http"
 	_ "net/http/pprof"
 	"os"
+	"os/user"
```

Comment:
I think you should use `github.com/mitchellh/go-homedir`, it works without the use of cgo, so the library can be used in cross-compilation environments

Reply:
I've made the change, but I doubt anyone is going to actually run this on MacOs. Is developer convenience worth adding yet another dependency which could be vulnerable to supply-chain attack? My personal view is that the whole DeFi ecosystem needs to be vastly more careful about whose code gets included.